estargz/estargz_test.go

```diff
@@ -107,6 +107,358 @@ func legacyFooterBytes(tocOff int64) []byte {
 	return buf.Bytes()
 }
 
+func TestWriteAndOpenWithTelemetry_AllHooksDefined(t *testing.T) {
```

Comment:
Could we combine `TestWriteAndOpenWithTelemetry_*` into a single test function?

Reply:
Yes, will update in the next revision of the PR.

Reply:
All set.